src/components/Composer.vue

```diff
@@ -335,7 +335,24 @@ export default {
 	},
 	computed: {
 		aliases() {
-			return this.$store.getters.accounts.filter((a) => !a.isUnified)
+			let cnt = 0
+			const aliases = []
+			const accounts = this.$store.getters.accounts.filter((a) => !a.isUnified)
+			accounts.forEach((account) => {
```

Comment:
I have the feeling something like `flatMap` would be more appropriate here. Which attributes of the account/alias do we need? Is a full deep copy necessary or can we just build new objects from the existing data?

Reply:
The selectedAlias which contains the selected value from the to field is used in 10 places in the composer. But you are right, not all data from the account is needed. Only the following attributes are needed: editorMode, signature, id, aliasId

Reply:
@ChristophWurst BTW flatmap has no IE support is that a problem?

Reply:
You can use https://lodash.com/docs/4.17.15#flatMap but we also have Babel so it should polyfill if flatMap is an ES standard